src/containers/meeting/third-party/index.tsx

```
@@ -7,58 +7,278 @@ import { useModalStore } from "@/store/modal";
+      router.replace('/meeting/third-party');
+    }
+  }, [searchParams, currentGroup]);
+
+  const loadGroupData = async () => {
+    try {
+      setLoading(true);
+
+      let groupStatus;
+      try {
+        groupStatus = await GroupService.getGroupStatus(client);
+      } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(@typescript-eslint/no-unused-vars)